src/test/java/com/ays/user/controller/UserControllerTest.java

```diff
@@ -17,12 +17,7 @@
 import com.ays.common.util.exception.model.AysErrorBuilder;
 import com.ays.user.model.User;
 import com.ays.user.model.UserBuilder;
-import com.ays.user.model.dto.request.UserListRequest;
-import com.ays.user.model.dto.request.UserListRequestBuilder;
-import com.ays.user.model.dto.request.UserSaveRequest;
-import com.ays.user.model.dto.request.UserSaveRequestBuilder;
-import com.ays.user.model.dto.request.UserUpdateRequest;
-import com.ays.user.model.dto.request.UserUpdateRequestBuilder;
+import com.ays.user.model.dto.request.*;
```

Comment:
Performans açısından bu kısmı aşağıdaki gibi düzeltebiliriz.
```suggestion
import com.ays.user.model.dto.request.UserListRequest;
import com.ays.user.model.dto.request.UserListRequestBuilder;
import com.ays.user.model.dto.request.UserSaveRequest;
import com.ays.user.model.dto.request.UserSaveRequestBuilder;
import com.ays.user.model.dto.request.UserUpdateRequest;
import com.ays.user.model.dto.request.UserUpdateRequestBuilder;
```
intellij idea'da Settings > Code Style > Java > Imports bölümünden aşağıdaki `import with '*':` ile biten alanları arttırırsan importları formatlamıyacaktır.

![image](https://github.com/afet-yonetim-sistemi/ays-be/assets/58088365/dfad3d5c-1c19-436b-9349-20cf84e0c634)

Reply:
Düzelttim, teşekkürler.